lib/parse/data/parseEmbedData.ts

```diff
@@ -25,10 +25,19 @@ const parseEmbedData = (config: IEmbedConfig, rssData?: IRss): IEmbedData => {
     title: rssTitle,
     link: rssShareUrl,
     image: rssImage,
+    podcast,
     itunes
   } = rssData || {};
   const { url: rssImageUrl } = rssImage || {};
   const { image: rssItunesImage, owner: rssItunesOwner } = itunes || {};
+  const { value: podcastValue } = podcast || {};
+  const podcastValueRecipient =
```

Comment:
only sets a payment pointer when the payment:value.type == 'webmonetization' and there is a valueRecipient within it. Always gets the top `valueRecipient`, an improvement would be for the player to handle multiple recipients, but that's not part of the initial scope.